src/main/java/catan/services/util/play/PreparationStageUtil.java

```diff
@@ -124,16 +120,16 @@ private boolean isEndOfCycle(GameBean game) {
         return firstPlayer && !oddCycle || lastPlayer && oddCycle;
     }
 
-    private GameUserActionCode getCurrentInitialAction(GameBean game) {
+    private GameUserActionCode getCurrentInitialBuildAction(GameBean game) {
```

Comment:
method returns END_TURN in some cases, so naming "InitialBuildAction" is incorrect